utils/shuffle_tables.c

```diff
@@ -0,0 +1,60 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <inttypes.h>
+
+#define extract(c,i) (3 & (c >> 2*i))
```

Comment:
The endianness of the code word is here; we should share this definition if possible.